volatility3/cli/__init__.py

```diff
@@ -231,6 +232,23 @@ def run(self):
             action="store_true",
         )
 
+
+        group = parser.add_mutually_exclusive_group()   
```

Comment:
Yep, as @eve-mem pointed out, adding global flags to the CLI options isn't really sustainable for every extra file system we want to support.  Python already provides the URL handler mechanism for adding new URLs, and as @eve-mem mentioned this is how we support the leechcore devices and would be how I recommend handling various buckets if they aren't as simple as http downloadable URLs.